### DISCUSSION TITLE
Offset not working as expected

### DISCUSSION THREAD

**Discussion by Xosted** [universam1/iSpindel#555] [Q&A]
Hello,

I have a problem with the offset calibration of two iSpindels.
After offset calibration and reboot, the measure tilt does not match.
The sequence of event is the following:
- Boot and connect to the web interface
- Navigate to the Info page.
- Tilt value: 85.8X
- Navigate the maintenance page
- Calibrate
- Tilt value: 89.8X
- Navigate to home page
- Power off then on
- Navigate to iSpindel page
- Tilt value: 85.8X

Sometimes, when checking the tilt the value would be static 54.74. It seems to happen randomly.
I have used 7.1.2 and 7.1.4.

I looked into the logs and I found that:
new offsets: 0:0:0
confirming offsets: 0:0:0
No offsets can explain why upon reboot there are no values to be loaded.
However, if there are no offsets, how come the tilt is corrected and maintained on reset?

Any idea what is happening?

**Comment by thegreatgunbantoad** [comment]
Weird things like this are often caused by clone ESP (Wemos) boards.  If your board has chamfered rather than rounded shoulders, it's a clone. Not all are bad, but some are iffy.  In which case replacing it may be simplest.

**Reply by thegreatgunbantoad** [reply]
There are also some odd accelerometers out there but they usually just fail to communicate completely.
